src/native.re

```diff
@@ -416,12 +441,13 @@ let finalOutputsScheme sortedSourcePaths::sortedSourcePaths => {
 
          -o: output file name.
          */
-      "ocamlfind ocamlc -custom %s %s -g -o %s %s %s 2>&1| berror; (exit ${PIPESTATUS[0]})"
+      "ocamlfind ocamlc -custom %s %s -g -open %s -o %s %s %s %s 2>&1| berror; (exit ${PIPESTATUS[0]})"
```

Comment:
We'll have to revisit the `Example` comment above this (not shown in this diff view). They're stale and misleading.